Kinetic_shape_reconstruction/doc/Kinetic_shape_reconstruction/Doxyfile.in

```diff
@@ -0,0 +1,7 @@
+@INCLUDE = ${CGAL_DOC_PACKAGE_DEFAULTS}
+
+PROJECT_NAME = "CGAL ${CGAL_DOC_VERSION} - Kinetic Shape Reconstruction"
+EXTRACT_ALL                =  NO
+HIDE_UNDOC_CLASSES         =  YES
+WARN_IF_UNDOCUMENTED       =  NO
+PREDEFINED                 =  DOXYGEN_NS
```

Comment:
Dangerous as it overrules the default settings, probably better to use `PREDEFINED += DOXYGEN_NS`

Reply:
Thank you, this was not intended.